package.json

```
@@ -2,36 +2,32 @@
   "name": "@github/textarea-autosize",
   "description": "Autosizes textarea to size of it's contents.",
   "version": "0.1.3",
-  "main": "dist/textarea-autosize.umd.js",
-  "module": "dist/textarea-autosize.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
```

This is potentially hazardous as the `dist/index.js` file uses es6 import/export. Many tools within the Node.js ecosystem will attempt to read this as a commonjs style export and likely fail.

Following my suggestion about having `tsc` bundle a `dist/umd/index.js` file, this could be changed to:

```suggestion
  "main": "dist/umd/index.js",
```